config/spryk/spryks/Spryker/Zed/Business/Validation/AddDomainEntityValidator.yml

```diff
@@ -27,28 +27,15 @@ arguments:
 
     targetFilename:
         value: "{{ domainEntity }}{{ type }}Validator.php"
+        
+    subDirectory:
+        value: "{{ domainEntity }}/Validator/"
 
     targetPath:
-        value: "src/{{ organization }}/Zed/{{ module }}/Business/Validator/{{ domainEntity }}/"
+        value: "src/{{ organization }}/Zed/{{ module }}/Business/"
 
 preSpryks:
     - AddModule
 
 postSpryks:
     - AddDomainEntityValidatorInterface
-    - AddZedBusinessFactoryMethod:
```

Comment:
Why this get removed?

Reply:
This method is unused. We use specific validators for create and update actions